resources/public/pamela.ebnf

```diff
@@ -191,13 +196,15 @@ FIELDS = <':fields'>
 GUARD = <':guard'>
 ICON = <':icon'>
 ID = <':id'>
+SAME = <'same'>
 IMPLIES = <'implies'>
 INFINITY = <':infinity'>
 INHERIT = <':inherit'>
 INITIAL = <':initial'>
 INTERFACE = <':interface'>
 LABEL = <':label'>
 LEAVE = <':leave'>
+ltm = <'LTM'>
```

Comment:
All of the existing Pamela reserved symbols are lowercase.  Is it important for `LTM` and `LM` to be uppercase? 
_Note that I'm asking that question since I don't know the meaning of these symbols, nor is it obvious based on the names_

Reply:
Fixed.  I have cleaned up the grammar to have reserved symbols in upper case with the other reserved symbols.  The non-terminal symbols are  with the other grammar rules in lower case. There is, this, a WM for the reserved symbol 'wm' and a grammar rule 'wm'.  Similarly for LTM and recency.

Reply:
I have also updated the PAMELA.md to reflect the changes described above and the fixed the section below that dealt with fields.  There is a case to be made for doing a more thorough cleanup of PAMELA.md but that should probably be filed as an issue.